Sources/Hackle/Common/HackleNotification.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  HackleNotification.swift
+//  Hackle
+//
+//  Created by sungwoo.yeo on 4/17/25.
+//
+
+import Foundation
+
+@objc public protocol HackleNotification {
+    var type: HackleNotificationClickActionType { get }
```

Comment:
```suggestion
    var actionType: HackleNotificationClickActionType { get }
```
Notification의 타입이라고 잘못 인지할수 있을거같습니다